### PR TITLE
Reduce blinking effects speed (300 -> 800ms), and add an option to change it in game

### PR DESCRIPTION
**Pull request by leoCottret**
#### Summary
SUMMARY: "Interface" Reduce blinking effects speed from 300 to 800ms, and add an option for change in game

#### Purpose of change
Blinking effects speed was 300ms, and could make people sensitive to high blink speed uneasy (most notably on the map).

#### Describe the solution
Change the blinking effects speed from 300 to 800ms.
Add an option to change it in game, since we all have our own idea of the ideal blinking effects speed.

#### Testing
Set the blink speed to the old 300ms
image

https://user-images.githubusercontent.com/71428793/200147650-3d3bb6f3-6170-4208-bcec-c6df94bf3487.mp4

Change the blinking effects speed from 300 to 800ms

https://user-images.githubusercontent.com/71428793/200147583-c29c37d9-149e-4a66-ab84-e6779a53b231.mp4

### CODE REVIEW

File: src/options.cpp [hunk 1]
```diff
@@ -1805,6 +1805,11 @@ void options_manager::add_options_graphics()
 
     get_option( "ANIMATION_DELAY" ).setPrerequisite( "ANIMATIONS" );
 
+    add( "BLINK_SPEED", "graphics", translate_marker( "Map blinking speed" ),
+         translate_marker( "The amount of time to pause between map extra informations blinking (notes, zombie hordes...) in ms." ),
```
Comment (olanti-p) [inline src/options.cpp:1809]: Name and description are not quite correct, this option affects multiple UIs, not only map screen.

Comment (leoCottret) [reply src/options.cpp:1809]: You're right, it also affects the creation of a zone, in the Zone manager, and possibly other things (I'm just trying to find some in game).
I changed it to
``` cpp
add( "BLINK_SPEED", "graphics", translate_marker( "Blinking effects speed" ),
         translate_marker( "The speed of every blinking effects in ms." ),
         100, 5000, 800
       );
```

**Comment by Coolthulhu** [comment]
```C++
 src/overmap_ui.cpp: In function ‘tripoint_abs_omt overmap_ui::display(const tripoint_abs_omt&, const overmap_ui::draw_data_t&)’:
Error: src/overmap_ui.cpp:1988:38: error: ‘BLINK_SPEED’ was not declared in this scope
 1988 |         action = ictxt.handle_input( BLINK_SPEED );
      |                                      ^~~~~~~~~~~
```

**Comment by Zireael07** [comment]
If this is not in DDA, it's well worth porting back.

**Pull request merged**